node/internal.ts

```diff
@@ -195,7 +195,7 @@ export function _setResourcePath(path: string, ignoreWarnings: boolean = false):
     }
     else {
         if (ignoreWarnings) {
-            _debug(_loc('LIB_ResourceFileAlreadySet', path));
+           // _debug(_loc('LIB_ResourceFileAlreadySet', path));
```

Comment:
please delete the line, no need to keep commented lines

Reply:
Hi @z00sts, I have removed the line.